### PR TITLE
ci: Merge C++17 build with one of the existing ones

### PR DESCRIPTION
**Pull request by maflcko**
No need to spin up an extra vm for each pull request for a simple sanity check that any of the other already running machines can test.

### CODE REVIEW

**Comment by laanwj** [comment]
Continues #18591 (ping @sipa).

**Comment by MarcoFalke** [comment]
Background is that the travis queue has a huge backlog. So we could either throw some more money at them, or (what seems simpler) not have that many configurations, most of which can be merged without any downside.

Screenshot_2020-04-30 Pull Requests - bitcoin bitcoin - Travis CI

File: .travis.yml [hunk 1]
```diff
@@ -148,11 +148,6 @@ jobs:
       env: >-
         FILE_ENV="./ci/test/00_setup_env_native_valgrind.sh"
 
-    - stage: test
-      name: 'x86_64 Linux  [GOAL: install]  [bionic]  [no depends, only system libs, c++17]'
```
Comment (sipa) [inline .travis.yml:152]: Maybe add a c++17 marker to the qt5 job it's merged with?

Comment (MarcoFalke) [reply .travis.yml:152]: Thanks, done

**Comment by sipa** [comment]
ACK

**Comment by MarcoFalke** [comment]
Still picked up: https://travis-ci.org/github/bitcoin/bitcoin/jobs/681586851#L3194

**Pull request merged**